### ISSUE TITLE
Disappoint 

### ISSUE THREAD

**Issue by edysegura** [mayconferreira/Payroll-Summary-Report#2]
@mayconferreira seu código é praticamente o mesmo que o código do @armgalison. 

Confiança é algo difícil de se construir e muito fácil de se quebrar.

![image](https://cloud.githubusercontent.com/assets/5324414/16359399/84561cfa-3b09-11e6-82cd-6db8e8564491.png)

**Comment by mayconferreira** [comment]
Sim, utilizei como base um código que rodou e infelizmente ao subirem para
o github, percebi que subiram o código errado. Percebi soh hoje de manhã ao
tentar ver o que fiz de errado. Pois na primeira prova utilizei o JQuery e
achei q você poderia zerar minha prova por este detalhe e no desespero ao
fazer a prova tentei adaptar ao meu código da prova anterior o javascript e
ao subir não percebi que a pessoa que subiu o código ao ir a prova do
roberto, subiu o errado. De qualquer forma você tem toda a razão. A dúvida
que fique de te enviar é esta de passar de JQuery para github que tive uma
boa dificuldade ao fazer o trabalho.

Att,

Maycon Ferreira




2016-06-25 19:20 GMT-03:00 Edy Segura <notifications@github.com>:

**Comment by mayconferreira** [comment]
Jquery para javascript*




Em 25 de junho de 2016 19:25, Maycon Ferreira <maycon.univas@gmail.com>
escreveu:
